### ISSUE TITLE
Hitting ESC should remove filter/go back to "all items"

### ISSUE THREAD

**Issue by raphmim** [padloc/padloc#271]
When browsing categories on desktop, it would be great if hitting ESC cleared the filters and led back to "all items". So that when searching for a particular item in a particular category, we could hit ESC once to clear the search field and a second time to clear the category, and we could immediately search for another item

**Comment by MaKleSoft** [comment]
I think the expected behavior for the escape key in this case is to exit the search, which is what it is doing right now. I totally understand the need to quickly search all items though and I was thinking about adding a dedicated keyboard shortcut for that:
```
⌘ + F => "regular" search, retaining the current context/filter
⌘ + ⇧ + F => "global" search, resetting the current context/filter to "all items"
```
Would that work for you?

**Comment by Raph33** [comment]
@MaKleSoft yes I think this would work and be an improvement. 

But I still think that from a UX point of view, sticking to the escape key would make it more accessible to all users. I also don't think it would go against the current use, it's more of extending the current use:

`if the user did a search item => esc clears the search field and goes back to all items`
`if the user filtered by category => esc clears the filter and goes back to all items`
`if the user selected an item => esc closes the item and goes back to the list of items`

So in practice:
`if I made a filtered search and then opened an item, pressing esc once will close the item, a second time will clear the search field, a third time will clear the filter, and I'll be back to the full list of items`
`if I made a regular search and then opened an item, pressing esc once will close the item, a second time will clear the search field, and I'll be back to the full list of items`

One of the issue is also that in the current set up, pressing escape only works if the search field is selected. So if I search a keyword, then click on an item, then close the item (either by pressing escape, clicking the back arrow, or clicking outside of the item), then pressing the escape key doesn't do anything.
